src/rules/enforce-memoize-async.ts

```
@@ -57,32 +58,77 @@ export const enforceMemoizeAsync = createRule<Options, MessageIds>({
+          fix(fixer) {
+            const fixes = [] as any[];
+
+            // Add import if it's not already present; ensure we only add once per file
+            if (!hasMemoizeImport && !scheduledImportFix) {
+              const sourceCode = context.getSourceCode();
+              const programBody = sourceCode.ast.body;
+              const firstImport = programBody.find(
+                (n) => n.type === AST_NODE_TYPES.ImportDeclaration,
+              );
+              const anchorNode = (firstImport ?? programBody[0]) as
+                | (typeof programBody)[number]
```

ESLint

[error] Replace `(typeof·programBody)` with `typeof·programBody`

(prettier/prettier)